packages/admin/cms-admin/src/dependencies/DependencyList.tsx

```
@@ -0,0 +1,229 @@
+
+                const loadUrl = () => {
+                    return dependencyObject.getUrl({
+                        rootColumnName: row.rootColumnName,
+                        jsonPath: row.jsonPath,
+                        contentScopeUrl: contentScope.match.url,
+                        apolloClient,
+                        id: row.id,
+                    });
+                };
+
+                return (
+                    <div style={{ display: "flex" }}>
+                        <IconButton
+                            onClick={async () => {
+                                const url = await loadUrl();
+                                window.open(url, "_blank");
+                            }}
+                        >
+                            <OpenNewTab />
+                        </IconButton>
+                        <IconButton
+                            onClick={async () => {
+                                const url = await loadUrl();
+
+                                history.push(url);
+                            }}
+                        >
+                            <ArrowRight />
+                        </IconButton>
+                    </div>
+                );
+            },
+        },
+    ];
+
+    let items: DependencyItem[] = [];
+    let totalCount = 0;
+
+    if (data?.item.dependencies) {
+        items = data.item.dependencies.nodes.map((node) => ({
+            ...node,
+            graphqlObjectType: node.targetGraphqlObjectType,
+            id: node.targetId,
+        }));
+        totalCount = data.item.dependencies.totalCount;
+    } else if (data?.item.dependents) {
+        items = data.item.dependents.nodes.map((node) => ({
+            ...node,
+            graphqlObjectType: node.rootGraphqlObjectType,
+            id: node.rootId,
+        }));
+        totalCount = data.item.dependents.totalCount;
+    } else if (!loading) {
+        console.error("Neither dependencies nor dependents is defined");
```

This should throw an error IMO

---

👍